tests/test_addons.py

```diff
@@ -485,6 +486,7 @@ def test_add_log_updates_cache_rename_via_move(self):
             'contentType': 'img/png'
         }).save()
 
+        update_storage_usage(self.node)
```

Comment:
nit: I'd move these into create version just to cut down on the number of new lines.